pkg/models/cedar_system_bookmark.go

```diff
@@ -10,3 +10,8 @@ type CedarSystemBookmark struct {
 	CedarSystemID string     `json:"cedarSystemId" db:"cedar_system_id"`
 	CreatedAt     *time.Time `db:"created_at"`
 }
+
+type BookmarkRequest struct {
```

Comment:
Can you leave a comment on this describing the purpose of the struct?

Reply:
yes! i was considering using the struct above it but opted not to because `CreatedAt` will be `nil` on a request, and I did not want an unpopulated field where someone may think it has data